exampleapp/src/main/res/raw/trillian_log.pem

```diff
@@ -0,0 +1,3 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
These can now be pulled directly from `v1/domain/info` for a simpler configuration